rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -405,6 +400,7 @@ def _linearly_increasing_batch_size(self, ep):
 
     def _train_tf(self, X, Y, intents_for_X,
```

Comment:
parameters should have types

Reply:
I missed this one on purpose because it goes over two lines.  I'll update it, but the sight of it makes me feel ill

Reply:
use style of types as in `__init__`